packages/babel-generator/test/fixtures/sourcemaps/ModuleExpression/source-map.json

```diff
@@ -11,5 +11,5 @@
   "sourcesContent": [
     "const m = module { export const foo = \"foo\" };\nmodule {\n  foo;\n  bar;\n};\nfoo(module {});"
   ],
-  "mappings": "AAAA,MAAMA,CAAC,GAAG;EAAS,OAAO,MAAMC,GAAG,GAAG,KAAK;AAAC,CAAC;AAC7C;EACEA,GAAG;EACHC,GAAG;AACL,CAAC;AACDD,GAAG,CAAC,SAAS,CAAC"
+  "mappings": "AAAA,MAAMA,CAAC,GAAG;EAAS,OAAO,MAAMC,GAAG,GAAG,KAAK;AAAC,CAAC;AAC7C;EACEA,GAAG;EACHC,GAAG;AACL,CAAC;AACDD,GAAG,CAAC,QAAQ,CAAC,CAAC"
```

Comment:
The source map is updated because we now generate two sections `module{` and `}` for the empty module expression `module{}`. Before this PR we map `module{}` as `module{}`.

Reply:
Maybe we could optimize this in the future (directly or in one of @jridgewell's packages) by merging source map segments that are adjacent both in the input and in the output.

Reply:
This shouldn't be technically difficult, but I'm not sure that's better. In the last PR I marked the sourcemap for every ";".

Reply:
This has to be done in Babel, because the sourcemap packages will not know whether this marking is important or not. Eg, `function foo() { return bar; }`, we should generate a marking for `bar` and `}` (the `}` is required for Chrome's debug stepping). I can't make the source map package ignore this `module {}` case without also ignoring that case.